client/src/components/Button/IconHOC.js

```diff
@@ -18,7 +22,6 @@ const IconHOC = (Component) => {
     icon: PropTypes.string
   };
 
-  IconComponent.defaultProps = Component.defaultProps;
```

Comment:
IconHOC is only used on Button.js. 

Button.defaultProps did not include className or icon, so removing this has no effect